src/theory/lemma_property.cpp

```diff
@@ -52,11 +52,14 @@ bool isLemmaPropertyNeedsJustify(LemmaProperty p)
 {
   return (p & LemmaProperty::NEEDS_JUSTIFY) != LemmaProperty::NONE;
 }
-
 bool isLemmaPropertyInprocess(LemmaProperty p)
 {
   return (p & LemmaProperty::INPROCESS) != LemmaProperty::NONE;
 }
+bool isLemmaPropertyLocal(LemmaProperty p)
+{
+  return (p & LemmaProperty::LOCAL) != LemmaProperty::NONE;
```

Comment:
Shouldn't this explicitly check for `(p & LemmaProperty::LOCAL) == LemmaProperty::LOCAL`? If `NONE` wouldn't be zero the above wouldn't work.

Reply:
`NONE` has to be zero; this check is doing the same as the other properties, are you suggesting we change all of them?

I don't have a strong opinion on this.

Reply:
Yes, I think it'd be cleaner to change the check and not rely on NONE being zero.

Reply:
Done